### PR TITLE
Adding Clique to MWIS 

### PR DESCRIPTION
**Pull request by maliheha**
- Extended the MWIS Mod, its documentation, and its tests to cover both maximum weighted independent set problem and its complement, maximum weighted clique problem
- Changed `mwis` to `mwis_mwc` everywhere
- Three doc-tests in Portfolio Mod fail because the presolved model size is slightly different in v11.0.0 compared to v10.0.3. I removed those three lines to allow the tests to pass because I think, in general, it is better not to base a test on the size of the presolved model. Let me know if you think we should address these failures in another way.

### CODE REVIEW

**Comment by maliheha** [comment]
Thanks @simonbowly! I reverted back the name changes and the changes in the portfolio documentation.

File: src/gurobi_optimods/mwis.py [hunk 1]
```diff
@@ -1,31 +1,69 @@
...
+        The total weight of the maximum weighted independent set (clique)
+    """
+
+    x: np.ndarray
+    f: float
+
+
+def check_input(adjacency_matrix, weights):
+    if not isinstance(adjacency_matrix, sp.spmatrix):
+        raise ValueError(
```
[the remark this answers is not in the document]
Comment (maliheha) [reply src/gurobi_optimods/mwis.py:36]: Good point! Done.

File: docs/source/mods/mwis.rst [hunk 2]
```diff
@@ -39,6 +45,9 @@ vertex :math:`i \in V` has a positive weight :math:`w_i`. Find a subset :math:`S
 * among all such independent sets, the set :math:`S` has the maximum total
   vertex weight.
 
+*Note*: In case all vertices have equal weights, the cardinality of
+set :math:`S` represents the stability number of graph :math:`G`.
```
[the remark this answers is not in the document]
Comment (maliheha) [reply docs/source/mods/mwis.rst:49]: No. This is just a definition in light of addressing the comments in issue #69. The ides is that: to find graph properties such as stability number and clique number, it is enough to solve the MWIS and clique problems where all vertices have weights 1 (or equal weights).

File: docs/source/conf.py [hunk 3]
```diff
@@ -81,6 +81,7 @@
     "Graph": "networkx.Graph",
     "LinAlgError": "numpy.linalg.LinAlgError",
     "spmatrix": "scipy.sparse.spmatrix",
+    "sparray": "scipy.sparse.sparray",
```
Comment (maliheha) [inline docs/source/conf.py:84]: Nice! Now I understand why the doc tests were failing.

**Comment by maliheha** [comment]
@simonbowly This is ready and supports networkx and pandas graphs as input as well.

**Comment by simonbowly** [comment]
Looks great, thanks! I just propose one minor change to the gurobipy-pandas formulation (7eab18a), can you please take a look? I think it's preferable to operate column-wise wherever possible.

**Pull request merged**